CHANGELOG.md

```diff
@@ -1,5 +1,50 @@
 # Changelog
 
+## 4.8.4 (2020-10-05)
+
+### Enhancements
+
+* Add device id to error reports on Cocoa platforms
+  [#203](https://github.com/bugsnag/bugsnag-unity/pull/203)
+
+* Update bugsnag-cocoa to v5.23.5:
+
+  * Fix JSON serialisation of strings with control characters
+    [#739](https://github.com/bugsnag/bugsnag-cocoa/pull/739)
+
+  * Removed non-thread safe date formatter
+    [#758](https://github.com/bugsnag/bugsnag-cocoa/pull/758)
+
+  * Fix incorrect string comparison of thread names in Mach exception handling
+    [#721](https://github.com/bugsnag/bugsnag-cocoa/pull/721)
+
+  * Move binary images store declaration from header file
+    [#725](https://github.com/bugsnag/bugsnag-cocoa/pull/725)
+
+  * Avoid dereference null pointer in JSON serialisation
+    [#637](https://github.com/bugsnag/bugsnag-cocoa/pull/637)
+    [Naugladur](https://github.com/Naugladur)
+
+  * Fix DYLD lock mechanism preventing compilation on iOS <10.
+    [#675](https://github.com/bugsnag/bugsnag-cocoa/pull/675)
+
+  * Fixed an issue where an app could deadlock during a crash if unfavourable 
+    timing caused DYLD lock contention.
+    [#580](https://github.com/bugsnag/bugsnag-cocoa/pull/580)
```

Comment:
I think we can condense the DYLD-related fixes as Unity users won't have had the intermediate states:

```suggestion
  * Avoid dereference null pointer in JSON serialisation
    [#637](https://github.com/bugsnag/bugsnag-cocoa/pull/637)
    [Naugladur](https://github.com/Naugladur)

  * Fixed an issue where an app could deadlock during a crash if unfavourable 
    timing caused DYLD lock contention.
    [#580](https://github.com/bugsnag/bugsnag-cocoa/pull/580)
    [#675](https://github.com/bugsnag/bugsnag-cocoa/pull/675)
    [#725](https://github.com/bugsnag/bugsnag-cocoa/pull/725)
    [#721](https://github.com/bugsnag/bugsnag-cocoa/pull/721)
```